<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VMeasureSetup</class>
 <widget class="QWidget" name="VMeasureSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1750</width>
    <height>730</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="cmbUnit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>670</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtX2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>180</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="txtX1">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>X1,Y1:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtX1">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>50</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbType">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="txtX1_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>X2,Y2:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>341</width>
     <height>661</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>338</number>
   </attribute>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="VDrawPlot" name="gvDraw">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>10</y>
     <width>710</width>
     <height>710</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblNow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>---</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtY1">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetF">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>360</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblType">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Measure Type:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lstFeature">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>471</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>610</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtY2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>310</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblUnit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>670</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Unit:</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VDrawPlot</class>
   <extends>QGraphicsView</extends>
   <header>vdrawplot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
